hr_applicant/models/hr_app_rel_edu.py

```diff
@@ -0,0 +1,157 @@
+from datetime import datetime
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.translate import _
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+
+class ApplicantRelative(models.Model):
+
+    _name = "applicant.relative"
+    _description = "Applicant Relatives"
+    _rec_name = "name"
+
+    relative_type = fields.Selection(
+        [
+            ("Aunty", "Aunty"),
+            ("Brother", "Brother"),
+            ("Daughter", "Daughter"),
+            ("Father", "Father"),
+            ("Husband", "Husband"),
+            ("Mother", "Mother"),
+            ("Sister", "Sister"),
+            ("Son", "Son"),
+            ("Uncle", "Uncle"),
+            ("Wife", "Wife"),
+            ("Other", "Other"),
+        ],
+        string="Relative Type",
+        required=True,
+    )
+    name = fields.Char(string="Name", size=128, required=True)
+    birthday = fields.Date(string="Date of Birth")
+    place_of_birth = fields.Char(string="Place of Birth", size=128)
+    occupation = fields.Char(string="Occupation", size=128)
+    gender = fields.Selection(
+        [("Male", "Male"), ("Female", "Female")], string="Gender", required=False
+    )
+    active = fields.Boolean(string="Active", default=True)
+    applicant_id = fields.Many2one(
+        "hr.applicant", "Applicant Ref", ondelete="cascade")
+
+    @api.onchange("birthday")
+    def _onchange_birthday(self):
+        if (
+            self.birthday
+            and fields.Date.from_string(self.birthday)>= fields.Date.today()
+        ):
+            warning = {
+                "title": _("User Alert !"),
+                "message": _("Date of Birth must be less than today!"),
+            }
+            self.birthday = False
+            return {"warning": warning}
+
+    @api.onchange("relative_type")
+    def _onchange_relative_type(self):
+        if self.relative_type:
+            if self.relative_type in ("Brother", "Father", "Husband", "Son", "Uncle"):
+                self.gender = "Male"
+            elif self.relative_type in ("Mother", "Sister", "Wife", "Aunty"):
+                self.gender = "Female"
+            else:
+                self.gender = ""
```

Comment:
remove else and define self.gender = "" at line 59

Reply:
yes mam, removed